docs/root/configuration/http_filters/rate_limit_filter.rst

```diff
@@ -68,22 +55,13 @@ the configuration.
 
 For the following configuration:
 
-.. code-block:: json
-
-  {
-    "actions" : [
-    {
-      "type" : "generic_key",
-      "descriptor_value" : "some_value"
-    },
-    {
-      "type" : "remote_address"
-    },
-    {
-      "type" : "souce_cluster"
-    }
-    ]
-  }
+.. code-block:: yaml
+
+  actions:
+      - {source_cluster: {}}
```

Comment:
This looks pretty strange YAML; did you validate?

Reply:
so if I tweak the rate limit test to dump MessageUtil::getJsonStringFromMessage( on a tweaked rate limit proto, I get
{
 "disable_key": "",
 "actions": [
  {
   "source_cluster": {}
  },
  {
   "remote_address": {}
  },
  {
   "generic_key": {
    "descriptor_value": "some_value"
   }
  }
 ]
}

which tossed into the first hit yaml converter online gets me this, which tossed into the first yaml *validator* online, thinks it's valid, but suggests

actions: 
  - 
    source_cluster: {}
  - 
    generic_key: 
      descriptor_value: some_value
I could switch to this, or I could fall back to json?

Reply:
We need to use YAML in all examples as this is the official textual representation of proto in Envoy.